### DISCUSSION TITLE
Turning off automatic id

### DISCUSSION THREAD

**Discussion by Duckbilled** [Zettlr/Zettlr#3294] [Q&A]
Is there a way to turn off the automatic file id? I want to use zettlr both as a zettelkasten as well as for other notes where I don't need the id. I have read one could use cmd + L for creating the ids manually.

**Comment by kyaso** [comment]
Yes, when you hit `Ctrl/Cmd + L`, an ID will be inserted in the text. Zettlr will look in the text for any string matching the ID regex, and the first it finds is taken as the ID of the file.

**Reply by Duckbilled** [reply]
Thank you for your quick reply. Is it also possible to turn off automatic generated ids as filenames (uid.md)?

**Reply by kyaso** [reply]
Under "Preferences -> Advanced" you can change the pattern for new filenames. Does that help?

**Reply by Duckbilled** [reply]
I checked that out, but then it creates a %uuid4.md. I guess I'll just rename and relocate those few notes I don't create for my Zettlekasten.
